lib/PuppeteerSharp/Page.cs

```diff
@@ -1603,6 +1603,13 @@ public Task SetBurstModeOffAsync()
 
             return Task.CompletedTask;
         }
+
+        /// <summary>
+        /// Brings page to front (activates tab).
+        /// </summary>
+        /// <returns>A task which resolves when the message was sent to Chromium.</returns>
```

Comment:
I think I would have written "that resolves" instead of "which resolves", but I'm not particularly against it, just a comment.

Reply:
Well, my curiosity led me to this:
https://www.writersdigest.com/online-editor/which-vs-that
so I think it needs to be changed to "that".

> If the sentence doesn’t need the clause that the word in question is connecting, use which. If it does, use that. (Pretty easy to remember, isn’t it?) Let me explain with a couple of examples